ethereum/aave/llama_aave_treasury/insert_aave_daily_treasury_events.sql

```diff
@@ -0,0 +1,272 @@
+CREATE OR REPLACE FUNCTION aave.insert_aave_daily_treasury_events(start_time timestamptz, end_time timestamptz) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+	start_time_day timestamptz := DATE_TRUNC('day',start_time);
+	end_time_day timestamptz := DATE_TRUNC('day',end_time) + interval '1 day'; --since we trunc to day
+BEGIN
+WITH rows AS (
+    INSERT INTO aave.aave_daily_treasury_events (
+contract_address,
+version,
+evt_day,
+difference,
+money_out_raw,
+money_in_raw,
+transfer_out,
+transfer_in,
+burn_out,
+mint_in,
+rewards_in,
+staking_out,
+staking_in,
+money_in,
+money_out,
+
+borrow_fees_originated,
+repay_fees,
+flashloan_v1_fees,
+flashloan_v2_fees,
+liquidation_fees,
+swap_fees,
+lend_burn_fees,
+deployer_in,
+other_fees,
+
+swap_out,
+swap_in,
+gas_out
+    )
+	
+WITH addresses AS (
+SELECT address FROM dune_user_generated.llama_treasury_addresses WHERE protocol = 'Aave' AND blockchain = 'Ethereum'
```

Comment:
`dune_user_generated.llama_treasury_addresses` this should be its own table. Things `dune_user_generated` can be removed and then this would break.

(Same applies for all occurrences of this table)